src/minutes/2019-10-02.md

```diff
@@ -0,0 +1,138 @@
+ttendees
```

Comment:
```suggestion
attendees
```

Reply:
Have pushed the fix. Thanks for catching this.